bessctl/conf/port/vhost/launch_container.py

```diff
@@ -99,6 +100,20 @@ def launch(cid):
     return proc
 
 
+def kill(cid):
+    print('Terminating container {} '.format(cid))
+
+    cmd = 'docker kill {name}'.format(name=CONTAINER_NAME + str(cid))
+
+    if VERBOSE:
+        print(cmd)
+
+    try:
+        proc = subprocess.check_call(shlex.split(cmd), stdout=subprocess.PIPE)
+    except subprocess.CalledProcessError:
+        pass
```

Comment:
Noticed in passing: you could just use `subprocess.call`, since we throw out the exception raised in `check_call` here.

Reply:
+ 1